css/style.css

```
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
+  padding: 1rem 0 1rem 0;
```

```suggestion
  padding: 1rem 0;
```
The shorthand is typically used, so we can do the same. Same goes for the rest of this file

---

That's awesome, I didn't think about doing shorthand on shorthand.